masonry_winit/examples/calc_masonry.rs

```diff
@@ -249,15 +249,15 @@ impl Widget for CalcButton {
 
     fn paint(&mut self, _ctx: &mut PaintCtx<'_>, _props: &PropertiesRef<'_>, _scene: &mut Scene) {}
 
-    fn accessibility_role(&self) -> Role {
-        Role::Button
+    fn accessibility_role(&self) -> accesskit::Role {
+        accesskit::Role::Button
```

Comment:
Any reason for this change? We usually go in the other direction.

Reply:
For the virtual scrolling, I chose to retain the prefix because the names are a bit oblique without the "`accesskit`" context. That being said, we should be consistent here, and reverting this is consistent with other versions.

Reply:
I'm fine with merging this as-is and deciding for a coding style guideline later.

Reply:
I concur - we can decide later, and we already have inconsistencies landed, so let's merge this